sites/public/pages/listing/[id]/[slug].tsx

```diff
@@ -15,11 +17,35 @@ interface ListingProps {
 export default function ListingPage(props: ListingProps) {
   const { listing } = props
 
+  const pageTitle = `${listing.name} - ${t("nav.siteTitle")}`
+  const { profile } = useContext(AuthContext)
+
+  useEffect(() => {
+    if (!listing.id) return
+    pushGtmEvent<ListingDetail>({
+      event: "pageView",
+      pageTitle: pageTitle,
+      status: profile ? USER_STATUS.LoggedIn : USER_STATUS.NotLoggedIn,
+      listingStartDate: listing.applicationOpenDate,
+      listingStatus: listing.status,
+      listingID: listing.id,
+      applicationDueDate: listing.applicationDueDate,
+      paperApplication: listing.paperApplication,
```

Comment:
Just curious why this one is on here?

Reply:
PaperApplication? Lori, our GA expert, thinks this is useful information to track.